node_modules/nforce-tooling/index.js

```diff
@@ -0,0 +1,449 @@
+var _ = require('lodash');
```

Comment:
I can reach out to @jeffdonthemic to see if we can get some PR's merged and released for `nforce-tooling`. I have collaborator access to the repo but not publish access in npm.